libs/ledger-live-common/src/hw/hooks/useGetLatestAvailableFirmware.test.ts

```diff
@@ -1,6 +1,7 @@
+import { of } from "rxjs";
 import { FirmwareUpdateContext } from "@ledgerhq/types-live";
 import { renderHook, act } from "@testing-library/react-hooks";
-import { of } from "rxjs";
+import { aLatestFirmwareContextBuilder } from "../../mock/fixtures/aLatestFirmwareContext";
```

Comment:
why is it aLatestFirmwareContextBuilder, what's the a?

Reply:
it's a naming convention i had before for mocking stuff: it's going to instantiate "a latestFirmwareContext" - the a means that the values themselves should not matter (compare to "the something"), and changing one of the prop (and it's associated `expect` checks) should not change the test

I like this naming convention